pages/_app.js

```diff
@@ -1,11 +1,14 @@
 import GlobalStyle from "../styles";
 import { GlobalProvider } from "@/components/GlobalProvider";
```

Comment:
are you using this anywhere in your app already? If not, please remove the import and the GlobalProvider.

Reply:
Sure. I'm happy to introduce this part further down the development path.